moon/config/moon-round1.json

```diff
@@ -47,7 +47,9 @@
               ["/qual_1_score", "srcp2_msgs/Qual1ScoringMsg"],
               ["/scout_1/volatile_sensor", "srcp2_msgs/VolSensorMsg"],
               ["/scout_1/camera/left/image_raw/compressed", "sensor_msgs/CompressedImage"],
-              ["/scout_1/camera/right/image_raw/compressed", "sensor_msgs/CompressedImage"]
+              ["/scout_1/camera/right/image_raw/compressed", "sensor_msgs/CompressedImage"],
+              ["/scout_1/imu", "sensor_msgs/Imu"],
```

Comment:
I see it is used now for all configs ...

Reply:
Yes, without it the node wouldn't be subscribed to any imu data